app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,38 @@
+class AccountActivationsController < ApplicationController
+  before_action :load_user_for_activation, only: [:edit]
+
+  def edit
```

Comment:
Thêm comment routes trên action của controller để tường minh hơn
```suggestion

  # GET: /account_activations/:id/edit
  def edit
```